Automattic/newspack-theme
# fix: improve appearance of shop page

### All Submissions:






### Changes proposed in this Pull Request:

This PR makes some general improvements to the shop landing page, including making sure it uses a consistent title style with other pages, and increases the grid of products to be 4 columns wide.

Some changes from the [product page PR ](https://github.com/Automattic/newspack-theme/pull/1271) will also affect the main shop page, including increasing the width of the content, and removing the breadcrumb and extra spacing from the top of the page. Ideally, this will be tested and merged after that one, so these changes can be tested with those. 

See #942

### How to test the changes in this Pull Request:

1. If you haven't already, set up a number of products through WooCommerce.
2. Apply this PR and run `npm run build`. 
3. View the /shop page on your site; confirm that:

* The products are displayed in four columns
* The shop title is title case (and not all uppercase), and is picking up the standard black page title colour, and not one of your custom colours.
* That there are no striking display issues.

### Other information:
